### ISSUE TITLE
Cannot find `arcgis/calcite-point-symbols` or `calcite-point-symbols` on NPM

### ISSUE THREAD

**Issue by yankuanz** [Esri/calcite-point-symbols#39]
I'm trying to install the library, but none of following is available on NPM:

- https://www.npmjs.com/package/@arcgis/calcite-point-symbols
- https://www.npmjs.com/package/arcgis/calcite-point-symbols
- https://www.npmjs.com/package/calcite-point-symbols

Am I missing something obviously here?

From the doc I see

- `npm install arcgis/calcite-point-symbols --save`
- `import { campground21 } from "calcite-point-symbols";`
- `import { campground21 } from "@esri/calcite-ui-icons/js/campground21.js";` which is not the point symbol library.

Thanks!

**Comment by sfrizzell** [comment]
@macandcheese can you speak to this/what needs to be done to resolve the issue?

**Comment by macandcheese** [comment]
The docs seem out of date / incorrect - cc @geospatialem - we should update this or remove the content.

As far as I know this isn't available on npm - maybe @paulcpederson has some insight there?

In the meantime - you should be able to install the dependency like

`"calcite-point-symbols": "github:esri/calcite-point-symbols.git#v2.2.5",`

**Comment by yankuanz** [comment]
Thanks @macandcheese

Confirmed that `"calcite-point-symbols": "github:esri/calcite-point-symbols.git#v2.2.5",` works for me.

---

Any plans to host it on NPM sometime? It will help with prototyping etc. Thanks

**Comment by macandcheese** [comment]
@sfrizzell - is this something your team would want to have on npm?

@cooney - is the above workaround sufficient for now?

**Comment by cooney** [comment]
@macandcheese yeah, the workaround is sufficient for us.  We are looking at including the point symbols in an upcoming release with StoryMaps.

It would be nice to update the documentation in case others are running into the same issue and wondering whats going on.
